packages/medusa-file-local/src/api/index.ts

```diff
@@ -0,0 +1,11 @@
+import express from "express"
+
+export default () => {
+  const app = express.Router()
+
+  const uploadDir = process.env.UPLOAD_DIR ?? "uploads/images"
```

Comment:
(Just stumbled upon this and thought I'd point it out 😄) Isn't it possible to access the plugin options in the exported function as a second parameter, and then get the value of `uploadDir` from `options.upload_dir` (similar to the constructor of the file service)? Basically what we have [here](https://docs.medusajs.com/development/endpoints/create#implementation).

Reply:
Yes indeed. Don't know why I missed that. Thanks @shahednasser 💪